src/e2e-test/java/io/cdap/plugin/dataplex/locators/DataPlexLocators.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.dataplex.locators;
+
+import io.cdap.e2e.utils.SeleniumDriver;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * PubSub Related Locators.
+ */
+public class DataPlexLocators {
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='plugin-Dataplex-batchsource']")
```

Comment:
Remove all unused locators.

Most of the  locators will not be required if proper step from e2e framework `PipelineSteps.java`
like `Enter input plugin property: "" with value: ""` `Select radio button plugin property: "" with value: ""` etc

Reply:
In this case all locators have been used from **PipelineSteps.java** so all dataplex locators can be removed.